test/e2e_node/e2e_service.go

```diff
@@ -208,6 +202,45 @@ func (es *e2eService) startServer(cmd *healthCheckCommand) error {
 	return fmt.Errorf("Timeout waiting for service %s", cmd)
 }
 
+func (es *e2eService) stopService(name string) error {
+	// Since node services are started with sudo, we must get the PID of the child process.
+	out, err := exec.Command("pidof", name).Output()
```

Comment:
That will signal all matching processes. In the event that multiple processes were running, I wanted to return an error and let the user handle it. In practice this probably won't happen, so I could go either way.